### ISSUE TITLE
Make Replace recovery codes keyboard accessible

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/fxa#598]
I noticed that our Settings has pretty excellent keyboard accessibilty. One of the few things I could find missing was this:
![inaccessible](https://user-images.githubusercontent.com/68213/46100261-f1c0ce00-c196-11e8-9244-148ab283ab7e.gif)
